config/locales/en.yml

```diff
@@ -81,6 +81,8 @@ en:
       weak_password: "To improve security you should change your login password to a more complex one"
     teams:
       not_existing: 'Es existiert kein Team mit der Id %{id}'
```

Comment:
This isn't your change but there's a german translation in english here? :thinking: